docs/src/interface.md

```diff
@@ -261,7 +264,8 @@ Order = [:macro, :type, :function]
 
 ## ValidationManifold
 
-[`ValidationManifold`](@ref) is a simple decorator that “decorates” a manifold with tests that all involved arrays are correct. For example involved input and output paratemers are checked before and after running a function, repectively.
+[`ValidationManifold`](@ref) is a simple decorator using the [`AbstractDecoratorManifold`](@ref) that “decorates” a manifold with tests that all involved arrays are correct.
```

Comment:
```suggestion
[`ValidationManifold`](@ref) is a simple decorator using the [`AbstractDecoratorManifold`](@ref) that “decorates” a manifold with tests that all involved points and vectors are valid for the wrapped manifold.
```
It also works when points are represented by more complicated things than an array.